src/bots/liquidator.ts

```diff
@@ -1331,7 +1426,26 @@ export class LiquidatorBot implements Bot {
 		const start = Date.now();
 		let ran = false;
 		try {
+			const usersCanBeLiquidated = new Array<{
+				user: User;
+				marginRequirement: BN;
+			}>();
 			for (const user of this.userMap.values()) {
+				const { canBeLiquidated, marginRequirement } = user.canBeLiquidated();
```

Comment:
you can remove the `canBeLiquidated` check couple lines down since you already do the check here